src/parser/jobs/brd/modules/SnapshotBlacklist.ts

```diff
@@ -0,0 +1,60 @@
+import STATUSES from 'data/STATUSES'
+
+export const SNAPSHOT_BLACKLIST = [
+	STATUSES.ARMYS_ETHOS.id,
+	STATUSES.ARMYS_MUSE.id,
+	STATUSES.ANGELS_WHISPER.id,
+	STATUSES.ARMS_LENGTH.id,
+	STATUSES.ASPECTED_HELIOS.id,
+	STATUSES.ASYLUM.id,
+	STATUSES.AURORA.id,
+	STATUSES.BARRAGE.id,
+	STATUSES.BLACKEST_NIGHT.id,
+	STATUSES.CATALYZE.id,
+	STATUSES.CLOSED_POSITION.id,
+	STATUSES.COLLECTIVE_UNCONSCIOUS_DIURNAL_MITIGATION.id,
+	STATUSES.COLLECTIVE_UNCONSCIOUS_NOCTURNAL.id,
+	STATUSES.CONFESSION.id,
+	STATUSES.COVERED.id,
+	STATUSES.DANCE_PARTNER.id,
+	STATUSES.DARK_MISSIONARY.id,
+	STATUSES.DIURNAL_BALANCE.id,
+	STATUSES.DIURNAL_INTERSECTION.id,
+	STATUSES.DIURNAL_OPPOSITION.id,
+	STATUSES.DIVINE_BENISON.id,
+	STATUSES.DIVINE_VEIL_PROC.id,
+	STATUSES.EVERLASTING_FLIGHT.id,
+	STATUSES.EXCOGITATION.id,
+	STATUSES.FEY_UNION.id,
+	STATUSES.GALVANIZE.id,
+	STATUSES.HEART_OF_LIGHT.id,
+	STATUSES.HEART_OF_STONE.id,
+	STATUSES.HOROSCOPE.id,
+	STATUSES.HOROSCOPE_HELIOS.id,
+	STATUSES.IMPROVISATION.id,
+	STATUSES.IMPROVISATION_HEALING.id,
+	STATUSES.INTERVENTION.id,
+	STATUSES.MANTRA.id,
+	STATUSES.MEDICA_II.id,
+	STATUSES.MEDITATIVE_BROTHERHOOD.id,
+	STATUSES.NASCENT_GLINT.id,
+	STATUSES.NATURES_MINNE.id,
+	STATUSES.NOCTURNAL_BALANCE.id,
+	STATUSES.NOCTURNAL_INTERSECTION.id,
+	STATUSES.NOCTURNAL_FIELD.id,
+	STATUSES.NOCTURNAL_OPPOSITION.id,
+	STATUSES.PASSAGE_OF_ARMS.id,
+	STATUSES.REGEN.id,
+	STATUSES.SACRED_SOIL.id,
+	STATUSES.SHAKE_IT_OFF.id,
+	STATUSES.SHIELD_SAMBA.id,
+	STATUSES.STRAIGHT_SHOT_READY.id,
+	STATUSES.TACTICIAN.id,
+	STATUSES.THE_WARDENS_PAEAN.id,
+	STATUSES.TROUBADOUR.id,
+	STATUSES.WEAKNESS.id,
+	STATUSES.WELL_FED.id,
+	STATUSES.WHEEL_OF_FORTUNE_DIURNAL.id,
+	STATUSES.WHEEL_OF_FORTUNE_NOCTURNAL.id,
+	STATUSES.WHISPERING_DAWN.id,
```

Comment:
Do you think it would have made more sense to make a whitelist instead?
We do kind of have a list of raidbuffs at `/parser/core/modules/RaidBuffs.js` so I would have thought it's possible to write a getter to pull these out.
This list will be unwieldy to maintain x_x